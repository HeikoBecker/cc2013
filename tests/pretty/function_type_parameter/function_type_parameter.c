/* Don't ask me why this compiles. I want to shoot the C standard developers
 * right now */
int f(int());
int g(int(*x)()){
	return 0;
}

