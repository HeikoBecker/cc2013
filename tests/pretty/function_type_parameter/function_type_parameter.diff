1c1
< int (f(int ()));
---
> int f(int );
