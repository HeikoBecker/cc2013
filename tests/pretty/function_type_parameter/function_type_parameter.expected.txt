int (f(int ()));

int (g(int ((*x)())))
{
	return 0;
}
